homeassistant/components/ecobee/repairs.py

```diff
@@ -27,11 +27,26 @@ def migrate_notify_issue(hass: HomeAssistant) -> None:
     )
 
 
+@callback
+def aux_heat_deprecation_issue(hass: HomeAssistant) -> None:
+    """Ensure an issue is registered."""
+    ir.async_create_issue(
+        hass,
+        DOMAIN,
+        "migrate_aux_heat",
+        breaks_in_ha_version="2024.10.0",
+        is_fixable=True,
+        is_persistent=True,
+        severity=ir.IssueSeverity.WARNING,
+        translation_key="migrate_aux_heat",
+    )
+
+
 async def async_create_fix_flow(
```

Comment:
The repair platform already provides this so why have we included it here?

Reply:
I wasn't able to find any issue creation specific to aux_heat deprecation in the repair platform. Is there an example I should model this after?